packages/light-apps/src/ManageAddresses/Add.tsx

```diff
@@ -3,21 +3,16 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import React from 'react';
-import { RouteComponentProps } from 'react-router-dom';
 import { Margin, Stacked, SubHeader } from '@substrate/ui-components';
 
 import { SaveAddress } from './SaveAddress';
 
-interface Props extends RouteComponentProps<{}> { }
-
-export class Add extends React.PureComponent<Props> {
-  render () {
-    return (
-      <Stacked>
-        <SubHeader> Enter an Address And Save it with A Name for Later Use. </SubHeader>
-        <Margin top />
-        <SaveAddress />
-      </Stacked>
-    );
-  }
+export function Add () {
+  return (
+    <Stacked>
+      <SubHeader> Enter an Address And Save it with A Name for Later Use. </SubHeader>
```

Comment:
Weird capitalization - "Enter an Address And Save it with A Name for Later Use."